crates/nu-cmd-lang/Cargo.toml

```
@@ -21,10 +21,10 @@ nu-protocol = { path = "../nu-protocol", version = "0.103.1", default-features =
 nu-utils = { path = "../nu-utils", version = "0.103.1", default-features = false }
 
 itertools = { workspace = true }
-shadow-rs = { version = "0.38", default-features = false }
+shadow-rs = { version = "1", default-features = false }
 
 [build-dependencies]
-shadow-rs = { version = "0.38", default-features = false }
+shadow-rs = { version = "1", features = ["build"] }
```

Looking at the features table in https://docs.rs/crate/shadow-rs/1.1.1/features and the new dependencies in `Cargo.lock` I am wondering if the following would suffice:
```suggestion
shadow-rs = { version = "1", default-features = false, features = ["build"] }
```

---

Ah, thank you for pointing out that, I just wondered why it introduces a new `git2` dependency, which we don't want.